microsoft-edge/extensions-chromium/getting-started/manifest-format.md

```diff
@@ -196,6 +197,25 @@ For reference information about each field, see [Manifest file format (V2)](http
 }
 ```
 
+---
+
+
+<!-- ====================================================================== -->
+## Changes in manifest.json from V2 to V3
+
+| V2 | V3 |
+|---|---|
+| `"manifest_version": 2,` | `  "manifest_version": 3,` |
+| `"action": ...,` | removed |
+| `"browser_action": {...},` | `"action": {...},` |
+| `"page_action": {...},` | `"action": {...},` |
+| `"persistent": false,` | removed |
+| If `background` is included, `service_ worker` is optional | If `background` is included, `service_ worker` is required |
+| `"host_permissions": ...,` | `"host_permissions": [...],` |
+| `"version_name": ...,` | `"version_name": "aString",` |
+
+
```

Comment:
@Bhuvana-Priya, tagging you here as I am unable to tag Siv. Please ask him to update this based on our discussion on comparing the MV2 and MV3 extension packages.

Reply:
@sivsouvamMSFT - can you update the table as discussed please? Thank you!

Reply:
I am assessing the [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/2101#pullrequestreview-1147435188) from Siv on main page of PR

Reply:
| V3 | V2 |
|---|---|
| `"manifest_version": 3,` | `  "manifest_version": 2 ,` |
| removed | "action": ...,` | 
| `"action": {...},` | `"browser_action": {...},` |
| `"action": {...},` | `"page_action": {...},` |
| removed | `"persistent": false,` |
| If `background` is included, `service_ worker` is required | If `background` is included, `service_ worker` is optional |
| `"host_permissions": [...],` | `"host_permissions": ...,` |
| `"version_name": "aString",` | `"version_name": ...,` |